default/python/AI/AIDependencies.py

```diff
@@ -78,7 +78,7 @@
 #  - [[EARLY_PRIORITY]]: Affected by species modifiers
 POP_SIZE_MOD_MAP_MODIFIED_BY_SPECIES = {
     # "Effect": [uninhab, hostile, poor, adequate, good]
-    "environment_bonus": [0,-4,-2, 0, 3],
+    "environment_bonus": [0, -4, -2, 0, 3],
```

Comment:
Oh, it was matrix structure.  I prefer to keep this changes and fix other lines of tables.